Complete Rework of the theme (Non-HTML)

tintinmaster opened the pull request:
Complete rework of the theme by hand. Added new .layout files of Steam.
Everything that can be designed by .layout and .style files should have the theme. (not the internal html sites)

I used a more decent looking Accent color from the dracula color palette. Accent colors are customizable by changing a variable in the steam.styles file. (README and INSTALL has to be updated)

Tested it only on Linux, will test it later on Windows.
It could be, that there are some bugs, because it was "trial and error" but i could not find any after deep searching


Edit: New Pull Request, because i used the dev branch before and not the master :facepalm:

Trollwut commented:
Hey @tintinmaster ! Thanks for the nice PR! :)

this is definitely an improvement to the now-master.

There are some bugs, yes. For example You chose pink as the "onClick"-color for buttons. But sometimes they're colored pink already when opening a window.

Also when a dropdown changed, it is also colored pink, which might be distracting.

We might need some additional work there, but I really like this approach.

Also you might want to consider adding yourself in the README.md in the team section :)

tintinmaster commented:
I now removed the colors from the buttons an dropdown menus, because the high contrast wasn't really fitting and I could not find a good looking setup. Onlything colorful are the notification buttons left to your account menu and are configurable through editing steam.styles
Also added myself to README.md as suggested

tintinmaster commented:
In der INSTALL.md müsste dann noch der Link aktualisiert werden für die dann neue ZIP-Datei.

Trollwut commented on INSTALL.md:
```diff
@@ -16,8 +16,8 @@ Use the terminal, Luke:
-3. move the extracted folder into your Steam skin folder (which is in the Steam base folder)
-   default location: C: -> Program Files -> Steam -> Skins
+3. move the extracted folder into your Steam skin folder (which is in the Steam base folder)  
+   default location: `C: -> Program Files -> Steam -> Skins`
```
we might want to write file paths as, well, file paths :) e.g. `C:/Program Files/Steam/Skins`

Trollwut commented on INSTALL.md:
```diff
@@ -1,31 +1,32 @@
 1. download or clone this repository
 2. extract (if downloaded compressed)
 3. move the extracted folder into your Steam skin folder (which is in the Steam base folder)
+   default location: C: -> Program Files -> Steam -> Skins
```
same here, we should write file paths as file paths :)

tintinmaster commented:
I just tested the theme on Windows. Everything working fine. Can't test on mac.

P.S. nice star wars reference in the INSTALL.md

Trollwut commented:
> I now removed the colors from the buttons an dropdown menus, because the high contrast wasn't really fitting and I could not find a good looking setup. Onlything colorful are the notification buttons left to your account menu and are configurable through editing steam.styles
> Also added myself to README.md as suggested

thanks for your work! I think I can test it tomorrow

> In der INSTALL.md müsste dann noch der Link aktualisiert werden für die dann neue ZIP-Datei.

thanks for reminding me, I already forgot that last update, whoopsie :D

tintinmaster commented:
My friend on a mac confirmed me that the theme is working properly on macOS

Trollwut commented:
Thanks mate!

Tested it on Linux and there seems to be no bug there - nice!

We still may talk about getting more colors into it, like more of the Dracula colors palette (I liked green for the new mesage stuff for example), but this is a very solid base. Thank you for your contribution!

tintinmaster commented:
I definitely want to add more color to the theme, bit it is hard deciding where to put color.

Unfortunate there is no easy way to modify the rest of Steam, without modifying the internal steam files. It is possible, but would probably break on every Steam Update :(

Trollwut commented:
I'd love to have it in the sidemenu for the selected game, like in the old screenshots.

But it seems theming has lost a lot of features with the new library update, just for the sake of having a browser into it...

Yes, breaking on updates has been a problem before, too, and it's a PIA.

zenorocha commented:
Great job on this! 🎉 🎉 🎉
